servers/physics/physics_server_sw.cpp

```diff
@@ -487,7 +487,8 @@ void PhysicsServerSW::body_set_space(RID p_body, RID p_space) {
 
 	while (body->get_constraint_map().size()) {
 		RID self = body->get_constraint_map().front()->key()->get_self();
-		ERR_CONTINUE(!self.is_valid());
+		if (!self.is_valid())
```

Comment:
Squashing in groups sound good, I let you see what groups make the most sense :) I don't mind if it's a bit verbose as those are important changes, but it could indeed be a bit less verbose.